packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/registries/registries.html

```diff
@@ -0,0 +1,207 @@
+                ></span>
+            </button>
+            <button
+                class="float-right oui-button oui-button_secondary oui-button_icon-only oui-button_s"
+                data-title="{{:: 'pci_ai_dashboard_private_docker_registries_data_refresh' | translate }}"
+                data-ng-click="$ctrl.refreshState()"
+            >
+                <span
+                    class="oui-icon oui-icon-refresh"
+                    aria-hidden="true"
+                ></span>
+            </button>
+        </oui-datagrid-topbar>
+    </oui-datagrid>
+
+    <oui-button
+        class="mb-3"
+        data-variant="secondary"
+        data-on-click="$ctrl.goToPrivateRegistry()"
+    >
+        <span
+            data-translate="pci_ai_dashboard_managed_private_registries_button"
+        ></span>
+    </oui-button>
+
+    <h4
+        class="mb-2"
+        data-translate="pci_ai_dashboard_shared_registries_title"
+    ></h4>
+    <oui-message type="warning">
+        {{:: 'pci_ai_dashboard_sared_registries_description'| translate }}
+    </oui-message>
+    <div class="mb-1">
+        <span
+            data-translate="pci_ai_dashboard_shared_registry_select_region"
+        ></span>
+        <label class="oui-select oui-select_inline" for="region">
+            <select
+                class="oui-select__input"
+                data-ng-model="$ctrl.currentRegion"
+                id="region"
+                data-ng-change="$ctrl.onRegionChange()"
+            >
+                <option ng-repeat="region in $ctrl.regions" ng-value="region"
+                    >{{region.id}}</option
+                >
+            </select>
+            <span
+                class="oui-icon oui-icon-chevron-down"
+                aria-hidden="true"
+            ></span>
+        </label>
+
+        <dl class="oui-description">
+            <dt>
+                Docker Registry
+                <button
+                    type="button"
+                    class="oui-popover-button"
+                    oui-popover="{{:: 'pci_ai_dashboard_shared_registry_help' | translate }}"
+                    oui-popover-placement="right"
+                ></button>
+            </dt>
+            <dd>{{ $ctrl.currentRegion.registryUrl }}</dd>
+            <dt
+                data-translate="pci_ai_dashboard_shared_registry_public_cloud_project"
+            ></dt>
+            <dd>{{ $ctrl.projectId }}</dd>
+        </dl>
+
+        <p
+            data-translate="pci_ai_dashboard_shared_registry_section_1_message_1"
+        ></p>
+        <div class="row d-lg-flex">
+            <div class="col-md-7">
+                <pre><code>docker login {{ $ctrl.currentRegion.registryUrl }}/{{$ctrl.projectId }}</code></pre>
+            </div>
+        </div>
+        <p class="mb-0">
+            {{:: 'pci_ai_dashboard_shared_registry_section_1_message_2' |
+            translate }}
```

Comment:
```suggestion
            <span data-translate="pci_ai_dashboard_shared_registry_section_1_message_2"></span>
```

Reply:
Hi @anooparveti, 
Yes, you're right. It has been modified!